themes/bootstrap3/js/lightbox.js

```diff
@@ -523,6 +523,13 @@ VuFind.register('lightbox', function Lightbox() {
     });
     _modal.on("shown.bs.modal", function lightboxShown() {
       bindFocus();
+
+      // Disable bootstrap-accessibility.js "enforceFocus" events.
+      // retainFocus() above handles it better.
+      // This is moot once that library (and bootstrap3) are retired.
+      var focEls = _modal.find(":tabbable");
+      var lastEl = $(focEls[focEls.length-1]);
```

Comment:
Looks like eslint wants spaces around the `-` in this line; not sure why the auto-generated message is getting attached to the wrong line of code when it displays here.

Reply:
Fixed